src/__tests__/__fixtures__/gh-474/spec.yaml

```diff
@@ -0,0 +1,14 @@
+openapi: 3.0.0
+info:
+  title: Test
+  version: 1.0.0
+paths:
+  '/':
+    get:
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: 'common.yaml#/components/schemas/ResponseType'
```

Comment:
Can we get another file identical to this with ./common.yaml too? We want to show it’s definitely working both ways.

Reply:
Yup, will do in a moment.